docs/_static/credential/v1.yaml

```diff
@@ -0,0 +1,246 @@
+        required: true
+        example:
+          - "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467"
+        schema:
+          type: string
+    post:
+      summary: "Revoke a credential"
+      operationId: "revoke"
+      tags:
+        - credential
+      responses:
+        "202":
+          description: Revocation has been accepted.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/VerificationOutcome"
+        "400":
+          description: Credential can't be revoked. Most likely due to a missing private key.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "404":
+          description: Credential is not found.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: An error occurred while processing the request.
+          content:
+            text/plain:
+              schema:
+                type: string
+  /internal/credential/v1/{concept}:
+    post:
+      summary: Search for a concept. A concept is backed by 1 or more VCs
+      operationId: "search"
+      tags:
+        - credential
+      requestBody:
+        required: true
+        description: a query with key value pairs and pagination
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/SearchRequest"
+      parameters:
+        - name: concept
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Found 0 or more results.
+          content:
+            application/json:
+              example:
+              schema:
+                type: array
+                items:
+                  description: the returned JSON depends on the requested concept.
+                  type: object
+        "400":
+          description: invalid params.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "404":
+          description: concept not found.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: An error occurred while processing the request.
+          content:
+            text/plain:
+              schema:
+                type: string
+components:
+  schemas:
+    CredentialSubject:
+      type: object
+      description: Contents of a Verifiable Credential giving name and DID of organization.
+    KeyValuePair:
+      type: object
+      description: used search params
+      required:
+        - key
+        - value
+      properties:
+        key:
+          description: >
+            Fields from VCs to search on. Concept specific keys must be prepended with the concept name and a '.'.
+            Default fields: issuer, subject, type
```

Comment:
Default fields don't need to be prepended with concept names? Maybe clear that up little bit